packages/typescript/test/ts_project/filegroup/a.ts

```diff
@@ -0,0 +1,5 @@
+export const a: string = 'hello';
+
+// Repro #2044
```

Comment:
I think you can simplify this test case a bit, no need to duplicate coverage we already have somewhere else. just a one-line TS file would work here

Reply:
Alright, I'll add a target to another package instead.